src/omen/cogs/wvw_commands.py

```diff
@@ -227,8 +306,11 @@ async def matchup(self, interaction: discord.Interaction):
                 )
                 embed_obj_value_string += f":{server}_circle: {camp_emoji}x{self.match_data_dict[server]['objectives']['camp']:=2} {tower_emoji}x{self.match_data_dict[server]['objectives']['tower']:=2} {keep_emoji}x{self.match_data_dict[server]['objectives']['keep']:=2} {castle_emoji}x{self.match_data_dict[server]['objectives']['castle']:=2}\n"
                 embed_skirmish_value_string += f":{server}_circle: Points: {self.match_data_dict[server]['skirmish']['skirmish_scores'][list(self.match_data_dict[server]['skirmish']['skirmish_scores'])[-1]]} (+{self.match_data_dict[server]['skirmish']['ppt']:=2} per tick)\n"
+                embed_predicted_victory_points_string += f":{server}_circle: Predicted Points: {self.match_data_dict[server]['predicted_victory_points']}\n"
             embed.add_field(name=f"Current Skirmish ({list(self.match_data_dict[server]['skirmish']['skirmish_scores'])[-1]} of 84)", value=embed_skirmish_value_string, inline=False)
             embed.add_field(name="Objectives", value=embed_obj_value_string, inline=False)
+            embed.add_field(name="Match Prediction", value=embed_predicted_victory_points_string, inline=False)
```

Comment:
![Screenshot 2023-01-29 114618](https://user-images.githubusercontent.com/35041/215341170-e902f3e3-34b6-4bac-8a21-e0e196cbab6f.png)

I ended up swapping the order of Objectives and Match Predictions after I took this screenshot, but this is what it looks like. Just predicts the final scores for each team. Could make it more explicit who wins, I guess.